email/src/main/java/org/hawkular/commons/email/EmailDispatcher.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.commons.email;
+
+import java.util.Map;
+
+import javax.mail.internet.InternetAddress;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+public interface EmailDispatcher {
+
+    boolean dispatch(InternetAddress recipient,
+                     String subject,
+                     String templatePlain,
+                     String templateHtml,
+                     Map<String, Object> properties)
+            throws Exception;
```

Comment:
Do we need the throws clause? It seems unused in the implementation. And if it's really needed, can we narrow down the exception type?